### ISSUE TITLE
Add LogLevel as additional Informatio

### ISSUE THREAD

**Issue by jfudickar** [exilon/QuickLogger#92]
Hi,
i would like to see something like a log level (as an integer) which could be used like a log stack information.
I want to use it for the following use cases:
- Increasing / Decreasing when entering / leaving a block.
So i can see clearly what happened in this block.
- Based on the log level I can then calculate automatically the duration from the last log call on the same level.
- I would also add an optional functionality to add blanks or \t before the log message based on the log level to allow a visualisation of the levels when reading the log.

Parts of that you can see in the CodeSite components. I didn't want to replace them but get some things out of this.

Do you think this could be helpful/usefull?
I could create a first Pull request for this.

Regards
Jens

**Comment by jfudickar** [comment]
Do you think it makes sense that I invest some time and create a pull request for that?

**Comment by exilon** [comment]
Are you talking about File provider?

**Comment by jfudickar** [comment]
No, I would add this to the core functionality, s9 that it can be used for all log providers

**Comment by jfudickar** [comment]
Any news?

**Comment by jfudickar** [comment]
???
I can provide a pull request for that, but I do not want to invest time if it's not wanted.

**Comment by exilon** [comment]
It must be an optional feature and don't alter normal flow of actual logger core. Quick.Debug implements a similar functionaliy getting elapsed time between a block or function.

**Comment by jfudickar** [comment]
I've started the first steps.

Can you have a look on it here : https://github.com/jfudickar/QuickLogger/tree/LogLevel
